src/hooks/useUrlHandler.ts

```diff
@@ -0,0 +1,10 @@
+import { Platform } from 'react-native'
+import { CustomTabs } from 'react-native-custom-tabs'
+import SafariView from 'react-native-safari-view'
+
+export const useUrlHandler = (url: string): (() => void) => {
```

Comment:
Why not use/extend/modify WebUtils? If this is the new way we should open URL's, then we ought to apply the change globally to the app and keep these utilities organized in one place.

Reply:
I don't think those utils use the same API to show an in-app browser. Are we not suppose to be using the in-app browser and instead `openBrowserUri`?